src/dialogs/more-info/controls/more-info-weather.ts

```diff
@@ -176,6 +274,9 @@ class MoreInfoWeather extends LitElement {
                               this.hass.locale,
                               this.hass.config
                             )}
+                            ${item.is_daytime !== false
+                              ? this.hass!.localize("ui.card.weather.day")
```

Comment:
You are returning `day` if `is_daytime === false`. Is that correct?

Reply:
I don't think that's the case, it returns `day` if `is_daytime !== false`?